noxfile.py

```diff
@@ -401,263 +401,6 @@ def _run_with_coverage(session, *test_cmd):
-@nox.parametrize('coverage', [False, True])
-def runtests_zeromq_m2crypto(session, coverage):
-    '''
-    runtests.py session with zeromq transport and m2crypto
-    '''
-    session.notify(
-        'runtests-parametrized-{}(coverage={}, crypto=\'m2crypto\', transport=\'zeromq\')'.format(
-            session.python,
-            coverage
-        )
-    )
-
-
-@nox.session(python=_PYTHON_VERSIONS, name='runtests-pycryptodomex')
-@nox.parametrize('coverage', [False, True])
-def runtests_pycryptodomex(session, coverage):
-    '''
-    runtests.py session with zeromq transport and pycryptodomex
-    '''
-    session.notify(
-        'runtests-parametrized-{}(coverage={}, crypto=\'pycryptodomex\', transport=\'zeromq\')'.format(
-            session.python,
-            coverage
-        )
-    )
-
-
-@nox.session(python=_PYTHON_VERSIONS, name='runtests-tcp-pycryptodomex')
-@nox.parametrize('coverage', [False, True])
-def runtests_tcp_pycryptodomex(session, coverage):
-    '''
-    runtests.py session with TCP transport and pycryptodomex
-    '''
-    session.notify(
-        'runtests-parametrized-{}(coverage={}, crypto=\'pycryptodomex\', transport=\'tcp\')'.format(
-            session.python,
-            coverage
-        )
-    )
-
-
-@nox.session(python=_PYTHON_VERSIONS, name='runtests-zeromq-pycryptodomex')
-@nox.parametrize('coverage', [False, True])
-def runtests_zeromq_pycryptodomex(session, coverage):
-    '''
-    runtests.py session with zeromq transport and pycryptodomex
-    '''
-    session.notify(
-        'runtests-parametrized-{}(coverage={}, crypto=\'pycryptodomex\', transport=\'zeromq\')'.format(
-            session.python,
-            coverage
-        )
-    )
-
-
-@nox.session(python=_PYTHON_VERSIONS, name='runtests-cloud')
-@nox.parametrize('coverage', [False, True])
-def runtests_cloud(session, coverage):
-    # Install requirements
-    _install_requirements(session, 'zeromq', 'unittest-xml-reporting==2.2.1')
-
-    pydir = _get_pydir(session)
-    cloud_requirements = os.path.join('requirements', 'static', pydir, 'cloud.txt')
-
-    session.install('--progress-bar=off', '-r', cloud_requirements, silent=PIP_INSTALL_SILENT)
-
-    cmd_args = [
-        '--tests-logfile={}'.format(RUNTESTS_LOGFILE),
-        '--cloud-provider-tests'
-    ] + session.posargs
-    _runtests(session, coverage, cmd_args)
-
-
-@nox.session(python=_PYTHON_VERSIONS, name='runtests-tornado')
-@nox.parametrize('coverage', [False, True])
-def runtests_tornado(session, coverage):
-    # Install requirements
-    _install_requirements(session, 'zeromq', 'unittest-xml-reporting==2.2.1')
-    session.install('--progress-bar=off', 'tornado==5.0.2', silent=PIP_INSTALL_SILENT)
-    session.install('--progress-bar=off', 'pyzmq==17.0.0', silent=PIP_INSTALL_SILENT)
-
-    cmd_args = [
-        '--tests-logfile={}'.format(RUNTESTS_LOGFILE)
-    ] + session.posargs
-    _runtests(session, coverage, cmd_args)
-
-
```

Comment:
Do I understand correctly that this is completely removing the "runtests" support?

Reply:
You understand correctly.